File: src/main/java/um/haberes/report/model/haberes/core/LiquidacionAdicionalDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LiquidacionAdicionalDto {

    private Long liquidacionAdicionalId;

    private Long legajoId;

    private Integer anho;

    private Integer mes;

    private Integer dependenciaId;

    private BigDecimal adicional = BigDecimal.ZERO;

    private PersonaDto persona;

    private DependenciaDto dependencia;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/LiquidacionAdicionalDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class LiquidacionAdicionalDto(

    var liquidacionAdicionalId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int? = null,
    var mes: Int? = null,
    var dependenciaId: Int? = null,
    var adicional: BigDecimal = BigDecimal.ZERO,
    var persona: PersonaDto? = null,
    var dependencia: DependenciaDto? = null

)
